### DISCUSSION TITLE
ERROR settings validation: server settings: listening address: getting unprivileged port start: opening file: open /proc/sys/net/ipv4/ip_unprivileged_port_start: no such file or directory

### DISCUSSION THREAD

**Discussion by jerryawade** [qdm12/ddns-updater#602] [General]
ERROR settings validation: server settings: listening address: getting unprivileged port start: opening file: open /proc/sys/net/ipv4/ip_unprivileged_port_start: no such file or directory
2024-01-19T10:34:06-06:00 INFO Shutdown successful

This error happened today after the latest container update. Before, everything working fine.

**Comment by GeorgesOrwell** [comment]
The same by me.

**Comment by qdm12** [comment]
Working on a fix sorry about that.

**Comment by qdm12** [comment]
12c46e7635a239025e999af82592d592b0bc09b5 should fix it, thanks for reporting the bug! (didn't bug on my desktop and server, it depended on the host it turns out).
